client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -286,28 +280,37 @@ export const AnalysisWizard: React.FC<IAnalysisWizard> = ({
   };
 
   const isModeValid = applications.every((app) => isModeSupported(app, mode));
+  const resetWizard = () => {
+    setCurrentTaskgroup(null);
+    reset();
+    onClose();
+  };
 
   const onSubmit = (fieldValues: AnalysisWizardFormValues) => {
     if (currentTaskgroup) {
       const taskgroup = setupTaskgroup(currentTaskgroup, fieldValues);
-
       submitTaskgroup(taskgroup);
     }
-    onClose();
+    resetWizard();
   };
 
   const onMove: WizardStepFunctionType = (
     { id, name },
     { prevId, prevName }
   ) => {
     if (id && stepIdReached < (id as number)) setStepIdReached(id as number);
+    if (id === 2) {
```

Comment:
What is 2? Any way we can use a step property or something instead?

Line 301 indicates that id is a string (otherwise why `as number`?), but line 302 is treating it as a number.

Reply:
Good point - we used to have an enum for this. I will create another one.

Reply:
![Screenshot 2023-08-08 at 2 54 07 PM](https://github.com/konveyor/tackle2-ui/assets/11218376/6fa656d7-6030-4231-8dcb-987f9eff0545)
Looks like the string/number type is coming from PF.